src/vrapp/VRVolumeApp.cpp

```diff
@@ -596,14 +635,14 @@ void VRVolumeApp::render(const MinVR::VRGraphicsState& renderState)
       ren->setClipping(false, nullptr);
   }
 
-  for (auto ren : m_renders) {
+  for (auto ren : m_renders)
```

Comment:
same question on iteration style here

Reply:
It's better to use foreach when no index is needed.